ui/src/Components/Company/Company.js

```diff
@@ -1,7 +1,175 @@
-export default function Company(){
-    return(
-    <div>
-        Company
-    </div>
+
+import { Link } from "react-router-dom";
+import { FiAirplay } from "react-icons/fi";
+import { FaListUl } from "react-icons/fa";
+import { BsFillGridFill } from "react-icons/bs";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { IoAddSharp } from "react-icons/io5";
+import './Company.css'
+
+
+
+
+import Card from "../Card/Card";
+
+
+const SERVER_URL = "https://trinitstechnologies.com/demo/api/v1/companies";
+function Company() {
+
+    const [showCard, setShowCard] = useState(false);
+    // let {isLoggedIn} = this.state;
+
+    const [companyModel, setcompanyModels] = useState([]);
+    const [Company, setCompanydata] = useState([]);
+    const [data,setData] = useState("");
+
+
+
+    function getData() {
+        const Url = "https://trinitstechnologies.com/demo/api/v1/companies";
+        axios.get(Url).then(response => setCompanydata(response.data)).catch((error) => {
+            alert("error is occured");
+        })
+
+    }
+    useEffect(() => {
+        getData();
+    }, []);
+
+
+    function editCompany(Company) {
+        alert(JSON.stringify(Company, null, 2));
+        const url = SERVER_URL + "/" + Company.id
+        axios.put(url)
+            .then(response =>
+                setCompanydata(prevModels => {
+                    const result = prevModels.slice();
+                    result.push(response.data);
+                    return result;
+                })
+
+                    .catch(error => alert('errr occuredd'))
+
+            )
+
+    }
+    function deletecompany(index) {
+        let value = window.confirm('are you sure delete this?')
+
+        if (value) {
+
+            let model = companyModel[index]
+            const url = SERVER_URL + "/" + model.id
+
+            axios.delete(url).then(response => response.data).catch(error => alert("error occured"))
```

Comment:
Toast a message when the user is successfully deleted and when the error occurred.